tests/network/test_continue_response.py

```diff
@@ -0,0 +1,599 @@
+        "network.beforeRequestSent", "cdp.Network.responseReceived",
+        "cdp.Network.loadingFailed"
+    ])
+
+    await send_JSON_command(
+        websocket, {
+            "method": "browsingContext.navigate",
+            "params": {
+                "context": context_id,
+                "url": hang_url,
+                "wait": "complete",
+            }
+        })
+
+    before_request_sent_event = await wait_for_event(
+        websocket, "network.beforeRequestSent")
+
+    # Assert these events never happen, otherwise the test is ineffective.
+    await assert_no_events_in_queue(
+        ["cdp.Network.responseReceived", "cdp.Network.loadingFailed"],
+        timeout=1.0)
+
+    assert not before_request_sent_event["params"]["isBlocked"]
+
+    network_id = before_request_sent_event["params"]["request"]["request"]
+
+    with pytest.raises(
+            Exception,
+            match=str({
+                "error": "no such request",
+                "message": f"No blocked request found for network id '{network_id}'",
+            })):
+        await execute_command(
+            websocket, {
+                "method": "network.continueResponse",
+                "params": {
+                    "request": network_id,
+                },
+            })
+
+
+@pytest.mark.parametrize("continueResponseParams", [
+    {
+        "headers": [{
+            "name": "header1",
+            "value": {
+                "type": "string",
+                "value": "value1",
+            }
+        }]
+    },
+    {
+        "statusCode": 401,
+    },
+],
+                         ids=["headers-only", "statusCode-only"])
+@pytest.mark.asyncio
+async def test_continue_response_must_specify_both_status_and_headers(
+        websocket, context_id, example_url, continueResponseParams):
+    await subscribe(websocket, ["network.beforeRequestSent"], [context_id])
+
+    await execute_command(
+        websocket, {
+            "method": "network.addIntercept",
+            "params": {
+                "phases": ["responseStarted"],
+                "urlPatterns": [{
+                    "type": "string",
+                    "pattern": example_url,
+                }, ],
+            },
+        })
+
+    await send_JSON_command(
+        websocket, {
+            "method": "browsingContext.navigate",
+            "params": {
+                "url": example_url,
+                "context": context_id,
+                "wait": "complete",
+            }
+        })
+
+    event_response = await wait_for_event(websocket,
+                                          "network.beforeRequestSent")
+    network_id = event_response["params"]["request"]["request"]
+
+    await subscribe(websocket, ["network.responseCompleted"])
+
+    # TODO(https://github.com/w3c/webdriver-bidi/issues/572): Follow up with spec.
+    with pytest.raises(
+            Exception,
+            match=str({
+                "error": "unknown error",
```

Comment:
Isn't it `Invalid argument` case?

Reply:
Please note the TODO above (https://github.com/w3c/webdriver-bidi/issues/572), which ponders exactly that. Feel free to make a comment in the upstream issue

Reply:
I tagged you there